packages/ferry_generator/pubspec.yaml

```
@@ -12,7 +12,12 @@ topics:
   - codegen
 dependencies:
   gql: '>=0.14.0 <2.0.0'
-  gql_code_builder: ^0.10.0
+  #gql_code_builder: ^0.10.1
+  gql_code_builder: 
+    git:
+      url: 'https://github.com/gql-dart/gql.git'
+      ref: 'bfafca57c04e5615bb38cd74f6e6b3e71d18994f'
+      path: ./codegen/gql_code_builder
```

```suggestion
  gql_code_builder: ^0.10.1
```

To make this PR as easy as possible: Accepting this change should work once a new version of gql_code_builder has been released